src/features/layers_panel/index.tsx

```diff
@@ -1,5 +1,14 @@
-import { MapLayerPanel } from './components';
+import { lazily } from 'react-lazily';
+import { i18n } from '~core/localization';
+import { MIN_HEIGHT } from './constants';
+import type { PanelFeatureInterface } from 'types/featuresTypes';
 
-export function MapLayersList() {
-  return <MapLayerPanel />;
-}
+const { PanelContent } = lazily(() => import('./components'));
+const { Layers24 } = lazily(() => import('@konturio/default-icons'));
```

Comment:
it doesn't seem to me like a good idea to import @konturio/default-icons lazily
just because we do it in a lot of parts of app not lazily